components/schema-migrator/migrations/provisioner/202119021531_drop_not_null_volume_size_gb_and_disk_type.down.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE gardener_config ALTER COLUMN disk_type SET NOT NULL;
```

Comment:
It'll also good to test how it behaves when there the `NULL` values in the column already. Are these deleted then? I mean when migrating backwards with the `...down.sql` migration

Reply:
True, I will add update with default values to down.sql migration

Reply:
Regarding transaction, I don't think it's necessary here. Those alterations are not closely related and don't really need to be performed together. However, it will be needed in down.sql when I add update with default values